wagtail_ab_testing/wagtail_hooks.py

```diff
@@ -66,28 +88,41 @@ def register_create_abtest_action_menu_item():
 
 # This is the only way to inject custom JS into the editor with knowledge of the page being edited
 class AbTestingTabActionMenuItem(ActionMenuItem):
-    def render_html(self, request, context):
-        if 'page' in context:
-            return format_html(
-                '<script src="{}"></script><script src="{}"></script><script>window.abTestingTabProps = JSON.parse("{}");</script>',
-                reverse('wagtail_ab_testing_admin:javascript_catalog'),
-                versioned_static('wagtail_ab_testing/js/wagtail-ab-testing.js'),
-                escapejs(json.dumps({
-                    'tests': [
-                        {
-                            'id': ab_test.id,
-                            'name': ab_test.name,
-                            'started_at': ab_test.first_started_at.strftime(DATE_FORMAT) if ab_test.first_started_at else _("Not started"),
-                            'status': ab_test.get_status_description(),
-                            'results_url': reverse('wagtail_ab_testing_admin:results', args=[ab_test.page_id, ab_test.id]),
-                        }
-                        for ab_test in AbTest.objects.filter(page=context['page']).order_by('-id')
-                    ],
-                    'can_create_abtest': request.user.has_perm('wagtail_ab_testing.add_abtest'),
-                }))
-            )
-
-        return ''
+    if WAGTAIL_VERSION >= (2, 15):
+        def render_html(self, context):
+            # v2.15 and later only requires the context object
+            if 'page' in context:
+                return self.format_html(context['request'].user, context)
+
+            return ''
+    else:
+        def render_html(self, request, context):
```

Comment:
suggestion: drop support for Wagtail <  2.15 as 2.15 is the LTS and everything before that is long not supported

Reply:
I like the suggestion. Saw you did this yesterday somewhere 😄

Reply:
Support is now for Wagtail >= 2.15